tests/get_or_create/tests.py

```diff
@@ -146,6 +146,25 @@ def test_defaults_exact(self):
         self.assertFalse(created)
         self.assertEqual(obj, obj2)
 
+    def test_callable_defaults(self):
+        """
+        If you pass a callable in the `defaults` kwarg,
```

Comment:
please make the line a bit longer (breaking near 79 characters) and add a period.